packages/caravan-wallets/vitest.config.ts

```diff
@@ -0,0 +1,15 @@
+import { defineConfig } from "vitest/config";
+
+export default defineConfig({
+  test: {
+    globals: true,
+    environment: "happy-dom",
+    alias: {
+      "bitbox-api": "vitest",
```

Comment:
interesting. this is just to mock it in during tests basically?

Reply:
wouldn't this undermine proper mocking during the tests? or can we still confirm the right methods are called?